src/networking.c

```diff
@@ -850,6 +850,15 @@ void addReplyDouble(client *c, double d) {
             addReplyProto(c, d > 0 ? ",inf\r\n" : ",-inf\r\n",
                               d > 0 ? 6 : 7);
         }
+    } else if (isnan(d)) {
```

Comment:
maybe we need to also patch ld2string?
for createStringObjectFromLongDouble and addReplyHumanLongDouble

Reply:
yes, the ld2string also need the patch, thanks for the tip. (btw, d2string also already normalize nan)